tests/lit-tests/vectorcall-2.ispc

```diff
@@ -0,0 +1,15 @@
+
+// RUN: %{ispc} %s -h %t1.h --target=host -o /dev/null
```

Comment:
``-o /dev/null`` is not needed.
Add ``--target-os=windows`` to make sure that it's compiled for Windows, otherwise it might fail in cross-compiler testing. Though right now we don't target Windows from non-Windows.